spec/services/bulk_upload/lettings/row_parser_spec.rb

```diff
@@ -531,6 +543,18 @@
         end
       end
 
+      context "when one of these fields is blank" do
```

Comment:
Would it make sense to have a more descriptive test name? I assume this is a compound validation that would otherwise set an error on each field?